src/main/java/seedu/agendum/ui/CommandBox.java

```diff
@@ -110,6 +113,25 @@ public void handle(KeyEvent event) {
             }
         });  
     }
+    
+    private void registerTabKeyEventFilter() {
+        commandTextField.addEventFilter(KeyEvent.KEY_PRESSED, new EventHandler<KeyEvent>() {
+            public void handle(KeyEvent event) {
+                KeyCode keyCode = event.getCode();
+                if (keyCode.equals(KeyCode.TAB)) {
+                    System.out.println("tab pressed");
```

Comment:
There is a println command executed in CommandBox.java too XD.
But autocomplete is nicely done. Easier to use now